sysl2/sysl/ValueUtil.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"github.com/anz-bank/sysl/src/proto"
+	"github.com/anz-bank/sysl/sysl2/sysl/utils"
```

Comment:
utils is not a separate package. it does not stand on its own.

Reply:
moved to main